actions.go

```diff
@@ -165,10 +165,12 @@ func (client *Client) CreateTriggerDefinition(input CustomActionsTriggerDefiniti
 		} `graphql:"customActionsTriggerDefinitionCreate(input: $input)"`
 	}
 	if input.AccessControl == nil {
-		input.AccessControl = RefOf(CustomActionsTriggerDefinitionAccessControlEnumEveryone)
+		everyoneEnum := CustomActionsTriggerDefinitionAccessControlEnumEveryone
+		input.AccessControl = &everyoneEnum
```

Comment:
Did you test if changing the enum to `var` instead of `const` makes is so we don't have to do this?

It should be a transparent change to the end user and make it so you can just do

```suggestion
		input.AccessControl = &CustomActionsTriggerDefinitionAccessControlEnumEveryone
```

Reply:
That does the trick. Should I open a separate PR to make this change to enum.go?

Reply:
Done ✅